### PR TITLE
Link GH issues

### PR DESCRIPTION
**Pull request by Amir-61**
* This PR links FA area labels with issues across all strongloop repositories (excluding scrum-loopback in strongloop-internal)

**Note:** While doing these changes, I noticed we are missing a few FA labels; @strongloop/loopback-dev please review this PR and if you see your FA is not linked, it means there was no label for your FA, so please make a label for your FA and update this PR accordingly.

Thanks

Related to https://github.com/strongloop/loopback-contributor-docs/pull/15#issuecomment-251119545

/to: @strongloop/loopback-dev @superkhau @bajtos

### CODE REVIEW

**Comment by Amir-61** [comment]
Please also note this PR is making FA labels linkable. I will do the same thing for each user in each FA as well, if you think that would be useful... So Please let me know if you think that would be useful as well or not?

**Comment by superkhau** [comment]
>so please make a label for your FA and update this PR accordingly.

Let's leave it up to the FA owners to update their own areas. Land yours and we'll land incremental updates instead of waiting for others to update your PR.

>Please also note this PR is making FA labels linkable. I will do the same thing for each user in each FA as well, if you think that would be useful

This is a good idea, I'm OK with this since it'll be easy to find issues to take care of.

**Comment by bajtos** [comment]
I personally find clickable headers non-intuitive. In many places, when one click on a header, the page URL is changed to point to that header. That way it's easy to get an URL for a specific section of a document.

I am proposing to use a different approach in this pull request and make it more explicit where the link is pointing to (i.e. to the list of open issues). I am proposing to place the link close to the GitHub label value. I edited the first FA section to show what I mean, see a69f1353.

**Comment by Amir-61** [comment]
>In many places, when one clicks on a header, the page URL is changed to point to that header. That way it's easy to get an URL for a specific section of a document.

👍  ✅ 

>I am proposing to place the link close to the GitHub label value.

Sure... I'll take care of it...

**Comment by Amir-61** [comment]
Thanks @bajtos  for feedback. I applied your feedback and I'm going to land the changes now.

File: functional-area-owners.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ See the actual content below for detailed examples.
 
 # API definition
 
+- GitHub label
+  - api-definition ([list of issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Aapi-definition)
```
Comment (davidcheung) [inline functional-area-owners.md]: missed close bracket

File: functional-area-owners.md [hunk 2]
```diff
@@ -49,25 +51,25 @@ See the actual content below for detailed examples.
...
   - [loopback-example-middleware](https://github.com/strongloop/loopback-example-middleware)
-- GitHub label
-  - boot
 
 # Components
 
 ## Passport
 
+- GitHub label
+  - [component-passport](https://github.com/issues?utf8=✓&q=is%3Aopen+label%component-passport)
```
Comment (davidcheung) [inline functional-area-owners.md]: should be `label%3Acomponent-passport`

**Comment by superkhau** [comment]
@Amir-61 Can you address @davidcheung's comments in a follow up PR?

**Comment by Amir-61** [comment]
Sure I'm doing now lol

**Comment by Amir-61** [comment]
Fixed with https://github.com/strongloop/loopback-contributor-docs/pull/18

Thanks for feedback!

**Pull request merged**